django/core/files/move.py

```diff
@@ -66,7 +66,13 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
+        # This will happen with copystat if the destination is a CIFS mount
+        if getattr(e, 'errno', 0) != 1:
```

Comment:
Errno 1 is "[Errno 1] Operation not permitted". It appears that PermissionError maps to EACCES and EPERM:

[PermissionError](https://docs.python.org/3/library/exceptions.html#PermissionError)

[Standard Errno Symbols](https://docs.python.org/3.5/library/errno.html)

@claudep Are you asking us to compare to Standard Errno Symbols if we can?